packages/core-design-system/package.json

```diff
@@ -10,7 +10,7 @@
     "./styles-esm": "./dist/styles-esm/index.ts"
   },
   "scripts": {
-    "build": " pnpm clean && pnpm build:styles",
+    "build:styles:2": "pnpm clean && pnpm build:styles",
```

Comment:
This is a temporary change to unblock the PR to merge. Once the script, it will be reverted.